plugins/parsers/json_v2/parser.go

```diff
@@ -80,6 +80,7 @@ func (p *Parser) Parse(input []byte) ([]telegraf.Metric, error) {
 
 	// Only valid JSON is supported
 	if !gjson.Valid(string(input)) {
+		p.Log.Debugf("Message: %s", input)
```

Comment:
Thanks for this, I think it will help, but I would ask that rather than having a debug message, you include this JSON as part of the error itself so anyone, whether they have debug enabled or not, can benefit.

This will make error messages longer with the additional data, but more useful and actionable.

Reply:
sure, happy to. PR updated - did you mean like that?